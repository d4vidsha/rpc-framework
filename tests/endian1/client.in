init ::1 1337
find add2_2
call add2_2 add2_2
55931 -1
close
