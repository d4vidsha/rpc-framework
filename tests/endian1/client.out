rpc_init_client: instance 0, addr ::1, port 1337
rpc_find: instance 0, add2_2
rpc_find: instance 0, returned handle for function add2_2
rpc_call: instance 0, calling add2_2, with arguments 55931 -1...
rpc_call: instance 0, call of add2_2 received result 55930
rpc_close_client: instance 0
